app/api_topology.go

```diff
@@ -79,17 +78,18 @@ func handleWebsocket(
 	renderer render.Renderer,
 	loop time.Duration,
 ) {
-	conn, err := upgrader.Upgrade(w, r, nil)
+	wsConn, err := upgrader.Upgrade(w, r, nil)
 	if err != nil {
 		// log.Info("Upgrade:", err)
 		return
 	}
+	conn := xfer.Ping(wsConn)
```

Comment:
Can we provide an xfer.Upgrade which automatically makes a pinger etc?  So you don't have to do this two-step everywhere.